registry/quilt_server/views.py

```diff
@@ -2307,6 +2317,134 @@ def _comment_dict(comment):
         contents=comment.contents
     )
 
+@app.route('/api/users/attach_role', methods=['POST'])
```

Comment:
This endpoint seems oddly named. If there's an `attach_role`, I'd expect a corresponding `detach_role`. On the other hand if you want a single endpoint to attach and detach, maybe `set_role` would be a better name.

Reply:
I switched this to set_roles in latest commit